src/database/tables/Quest.entity.ts

```diff
@@ -86,6 +86,17 @@ registerEnumType(QuestPeriod, {
   name: 'QuestPeriod',
 });
 
+export enum QuestType {
+  GENESIS = 'GENESIS',
+  PRIME = 'PRIME',
+  DAILY = 'DAILY',
+  PROGRESSIVE = 'PROGRESSIVE',
+}
+
+registerEnumType(QuestType, {
+  name: 'QuestType',
+});
+
```

Comment:
can you move this enum under the `src/database/enums` folder? There is currently an issue with typeorm where it will incorrectly try to register the enums multiple times if this file is ever imported into another `database/` file which causes the migrations to fail

Reply:
I moved QuestType to enums folder